src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -87,16 +87,16 @@ public static String sanitizeTextField(String rawText) {
     public static Text sanitizeTextField(Text rawText) {
         return (rawText==null) ? null :  new Text(trimIfNotNull(rawText.getValue()));
     }
-    
-    public static String sanitizeHtmlForSaving(String html) {
-        return sanitizeForHtml(html);
-    }
 
     /**
      * Escape the string for inserting into javascript code.
-     * This automatically calls {@link #escapeHTML} so make it safe for HTML too.
+     * This automatically calls {@link #sanitizeForHtml} so make it safe for HTML too.
+     *
+     * @param string
+     * @return the sanitized string or null (if the parameter was null).
      */
     public static String sanitizeForJs(String str){ 
+        if(str == null) return null;
```

Comment:
Please refer to our Java coding style guide at https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true

For your ease -

![screen shot 2016-03-22 at 8 41 32 pm](https://cloud.githubusercontent.com/assets/6291947/13975026/869eb656-f06e-11e5-8c82-98a84654ed4b.png)

Please fix